composer.json

```diff
@@ -92,6 +92,7 @@
 
         "npm-asset/sprintf-js":     "~1.0.0",
         "npm-asset/codemirror":     "~5.23.0",
+        "npm-asset/notifyjs":       "*",
```

Comment:
You should use a more specific version of the library.

Reply:
Okay will do.